demo/nodejs/mic.js

```diff
@@ -90,7 +90,16 @@ async function micDemo() {
     const pcm = await recorder.read();
     try {
       const voiceProbability = engineInstance.process(pcm);
-      console.log(voiceProbability);
+      const percentage = voiceProbability * 100;
```

Comment:
when you try the demo without an access key it tells you 'no access key provided', but doesn't give you the usage instructions e.g. `-a, --access_key`